db/tests/unit/events.rs

```diff
@@ -991,6 +991,24 @@ fn genres() {
     );
 }
 
+#[test]
+fn find_all_ticket_holders_count() {
+    let project = TestProject::new();
+    let connection = project.get_connection();
+    let user = project.create_user().finish();
+    let event = project.create_event().with_ticket_pricing().finish();
+    project
+        .create_order()
+        .for_user(&user)
+        .for_event(&event)
+        .quantity(5)
+        .is_paid()
+        .finish();
+
+    let ticket_holders = Event::find_all_ticket_holders_count(event.id, connection).unwrap();
+    assert_eq!(ticket_holders, 5);
```

Comment:
```suggestion
    assert_eq!(ticket_holders, 1);
```
There is only one ticket holder here. They have 5 tickets